metadata-ingestion/src/datahub/api/circuit_breaker/assertion_circuit_breaker.py

```diff
@@ -12,6 +12,9 @@
 from datahub.api.graphql import Assertion, Operation
 
 logger: logging.Logger = logging.getLogger(__name__)
+from gql.transport.requests import log as requests_logger
+
+requests_logger.setLevel(logging.WARNING)
```

Comment:
Can you move this to the abstract class' init and then this change would be available for all the circuit breakers.